src/test/java/org/opentripplanner/updater/trip/RealtimeTestEnvironment.java

```diff
@@ -221,6 +221,7 @@ public TripPattern getPatternForTrip(Trip trip) {
   }
 
   public TimetableSnapshot getTimetableSnapshot() {
+    commitTimetableSnapshot();
```

Comment:
In which test case is this necessary?

Reply:
This is the wrong place! It should be called only when applying GTFS trip updates and SIRI estimated tables.
Fixed.